Console/retro/ConsoleWindow.cc

```diff
@@ -35,8 +35,8 @@ namespace
 ConsoleWindow::ConsoleWindow(Rect r, ConstStr255Param title)
 {
     GrafPtr port;
-
-    win = NewWindow(NULL, &r, "\pRetro68 Console", true, 0, (WindowPtr)-1, false, 0);
+        //Retro68 Improved Console
+    win = NewWindow(NULL, &r, "\pThe Queen's Footsteps", true, 0, (WindowPtr)-1, true, 0);
```

Comment:
I definitely have to try playing The Queen's Footsteps on one of my old Macs (an emulator just wouldn't be the same).
It's a bit confusing as a default console title, though :-)

Reply:
You are right!!!

Sorry, I forgot to edit that.

BTW, I think we would need a way to let the program change the title of the console window.

Reply:
This is something I forgot to edit, sorry!

I think something is needed so that the program can set the title of the console window.

BTW, I hope you'll enjoy the game! :-)

Reply:
Guess what... there's an escape sequence for that!
A non-standard one, but a widespread one, popularized by `xterm`:
https://en.wikipedia.org/wiki/ANSI_escape_code#Escape_sequences
(under Operating System Command)

Another possible future feature.

Reply:
That's really great! Is it easy to change the name of a window on the fly, by the way?